src/test/java/org/jabref/logic/layout/LayoutEntryTest.java

```diff
@@ -95,4 +103,44 @@ void parseMethodCalls() {
         assertEquals("test", (LayoutEntry.parseMethodsCalls("bla(test),foo(fark)").getFirst()).get(1));
         assertEquals("fark", (LayoutEntry.parseMethodsCalls("bla(test),foo(fark)").get(1)).get(1));
     }
+
+    private void assertUnsupportedOperation(int type, List<StringInt> parsedEntries, BibDatabaseContext context) {
+        LayoutEntry layoutEntry = new LayoutEntry(parsedEntries, type, null, null, null);
+        assertThrows(UnsupportedOperationException.class, () -> layoutEntry.doLayout(context, StandardCharsets.UTF_8));
+    }
+
+    private void assertLayoutResult(int type, String expected, List<StringInt> parsedEntries, BibDatabaseContext context) {
+        LayoutEntry layoutEntry = new LayoutEntry(parsedEntries, type, null, null, null);
+        assertEquals(expected, layoutEntry.doLayout(context, StandardCharsets.UTF_8));
+    }
+
+    @ParameterizedTest
+    @CsvSource({
+            "2",
+            "3",
+            "4",
+            "6",
+            "7"
+    })
+    void testUnsupportedOperationTypes(int type) {
+        List<StringInt> parsedEntries = List.of(new StringInt("place_holder", 0),
+                new StringInt("testString", 0));
+        BibDatabaseContext bibDatabaseContext = new BibDatabaseContext(new BibDatabase(), new MetaData(), null);
+        assertUnsupportedOperation(type, parsedEntries, bibDatabaseContext);
+    }
+
+    @ParameterizedTest
+    @CsvSource({
+            "1, testString",
+            "5, testString",
+            "8, UTF-8",
+            "9, ''",
+            "10, ''"
+    })
+    void testLayoutResult(int type, String expectedValue) {
```

Comment:
I wonder why OpenRewrite does not state anything (https://docs.openrewrite.org/recipes/java/testing/cleanup/removetestprefix)

--> test names should not start with `test` just remove `test` and strt with a lower case `L`.

Reply:
> I wonder why OpenRewrite does not state anything

Because it seems there is a conflict with `main`, the OpenRewrite didn't start to work in this situation.